### PR TITLE
Push handler to Syslog

### PR DESCRIPTION
**Pull request by srleach**
## Features
- Prevent logs from being pushed using Loggly handler, and insert into syslog instead. 

https://www.pivotaltracker.com/story/show/112319685

### CODE REVIEW

File: bootstrap/ConfigureLogging.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    /**
+     * Configure the Monolog handlers for the application.
+     *
+     * @param  \Illuminate\Contracts\Foundation\Application  $app
+     * @param  \Illuminate\Log\Writer  $log
+     * @return void
+     */
+    protected function configureSingleHandler(Application $app, Writer $log)
+    {
+        $syslogHandler = new SyslogHandler(env('LOG_PREFIX', 'BasketLog'), LOG_USER, LOG_NOTICE);
```
Comment (srleach) [inline bootstrap/ConfigureLogging.php]: Are we sure we want to push this with severity `LOG_NOTICE` ?

**Comment by EvanBarbour3** [comment]
@srleach Doesn't seem to be logging to /var/log/syslog ..

**Comment by EvanBarbour3** [comment]
Turns out it does work, wasn't looking in the right place for the log

**Comment by srleach** [comment]
To clarify for others in the PR, the changes __do__ push logs into /var/log/syslog

**Comment by wnowicki** [comment]
@EvanBarbour3 has this been reviewed ?

**Comment by wnowicki** [comment]
@EvanBarbour3 :question: :bomb:

**Pull request merged**